lib/code_corps_web/views/task_list_view.ex

```diff
@@ -7,5 +7,5 @@ defmodule CodeCorpsWeb.TaskListView do
 
   has_one :project, type: "project", field: :project_id
 
-  has_many :tasks, serializer: CodeCorpsWeb.TaskView, identifiers: :always
+  has_many :tasks, serializer: CodeCorpsWeb.TaskIncludedView, identifiers: :always, include: true
```

Comment:
Maybe better to call this TaskSlimView.

Reply:
Not a bad idea. Ideally, I would love if we could just specify what to include and where.

Possibly even the Ember client would do it, so we don't have to worry about this.

Reply:
kind of like graphql?  I would love that.  I worked on a project and eventually there was 5 different views for the same resource, each giving you a different subset of data.  It was slightly hard to manage.

We implemented the request on the ember side, but eventually the logic got complex and the various views were easier to manage.